prov/efa/test/efa_unit_test_info.c

```diff
@@ -77,8 +77,8 @@ void test_info_open_ep_with_api_1_1_info()
 	 * "mode". For example, the requirement of local memory registration
 	 * is specified as FI_LOCAL_MR.
 	 */
-	hints->mode = FI_LOCAL_MR;
-	hints->domain_attr->mr_mode = FI_MR_BASIC;
+	hints->mode = OFI_LOCAL_MR;
```

Comment:
We shouldn't use `OFI_*` mode here as `OFI_*` is intended for internal usage without exposure to applications. The blame shows this test was added as part of https://github.com/ofiwg/libfabric/pull/7873 and the PR shows we used to handle `FI_LOCAL_MR` and `FI_MR_BASIC` in the efa provider code specially. It's not clear to me why these mode bits don't show in efa provider now. Can you chase it down?

Reply:
It was removed in https://github.com/ofiwg/libfabric/commit/613e69e9bc46208ce22dd1cdd9f3c59079b6b6a3

Reply:
Do we leave it as it is then?

Reply:
Just remove it as the tested code path doesn't exist after https://github.com/ofiwg/libfabric/commit/613e69e9bc46208ce22dd1cdd9f3c59079b6b6a3